### PR TITLE
Eth

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: backend/devices/bitbox/keystore.go [hunk 1]
```diff
@@ -75,11 +77,18 @@ func (keystore *keystore) ExtendedPublicKey(
...
+		return keystore.SignBTCTransaction(specificProposedTx)
+	case *eth.TxProposal:
+		return keystore.SignETHTransaction(specificProposedTx)
+	default:
+		panic("unknown proposal type")
 	}
-	keystore.log.Info("Sign transaction")
+}
+
+func (keystore *keystore) SignBTCTransaction(btcProposedTx *btc.ProposedTransaction) error {
```
Comment (KasparEtter) [inline backend/devices/bitbox/keystore.go]: Make method private. (And personal preference: Avoid forward reference in `SignTransaction`, i.e. move this method above `SignTransaction` – but we're not consistent anyway and I can live without such a change.)

File: backend/devices/bitbox/keystore.go [hunk 2]
```diff
@@ -132,3 +141,31 @@ func (keystore *keystore) SignTransaction(proposedTx coin.ProposedTransaction) e
 	}
 	return nil
 }
+
+func (keystore *keystore) SignETHTransaction(txProposal *eth.TxProposal) error {
```
Comment (KasparEtter) [inline backend/devices/bitbox/keystore.go]: Make method private. (And personal preference: Avoid forward reference in `SignTransaction`, i.e. move this method above `SignTransaction` – but we're not consistent anyway and I can live without such a change.)

File: backend/devices/bitbox/keystore.go (hunk 1 above)
Comment (benma) [reply backend/devices/bitbox/keystore.go]: This is not C :P Done :)

File: backend/devices/bitbox/keystore.go (hunk 2 above)
Comment (benma) [reply backend/devices/bitbox/keystore.go]: Done.

**Pull request merged**